projects/bp-gallery/src/components/views/location-curating/tag-structure-helpers.tsx

```diff
@@ -192,3 +203,43 @@ export const useGetBreadthFirstOrder = (
 
   return tagOrder;
 };
+
+export const useGetChildMatrix = (flattenedTags: FlatTag[] | undefined) => {
```

Comment:
Maybe rename this to something like `useDescendantsMatrix`, since it includes transitive relationships. If you do so, also rename the other mentions of child/children in this context.